include/cantera/transport/TransportBase.h

```diff
@@ -585,6 +585,41 @@ class Transport
         throw NotImplementedError("Transport::getMixDiffCoeffsMass");
     }
 
+    //! Get/set the polynomial fits to the transport properties
```

Comment:
Replace with main docstring (see other comment). Same for other functions below.

Reply:
OK